space-plugins/nuxt-base/server/middleware/02.auth.global.ts

```diff
@@ -13,6 +13,18 @@ export default defineEventHandler(async (event) => {
 		return;
 	}
 
+	// Delete cookie and initiated OAuth flow
+	// if the user hasn't been authenticated yet.
+	// (Storyfront attaches this query parameter in that case)
+	if (getQuery(event)['init_oauth'] === 'true') {
+		setCookie(event, 'sb.auth', '', {
+			httpOnly: true,
+			secure: true,
+			sameSite: 'none',
+		});
+		return await sendRedirect(event, appConfig.auth.initOauthFlowUrl, 302);
```

Comment:
We send user to the oauth page here, but I've kept the line 36, because we still need it.

- line 25: for the initial oauth flow (not granted yet)
- line 36: everytime user opens the plugin (after being granted)